python/libcudf/CMakeLists.txt

```diff
@@ -58,3 +61,12 @@ if(USE_NVCOMP_RUNTIME_WHEEL)
     APPEND
   )
 endif()
+
+if(USE_LIBKVIKIO_RUNTIME_WHEEL)
+  set(rpaths "$ORIGIN/../../libkvikio/lib64")
```

Comment:
Instead of using RPATH we should call the load function. I'll update that locally and push.